pkg/master/master.go

```diff
@@ -94,7 +94,7 @@ const (
 	// DefaultEndpointReconcilerTTL is the default TTL timeout for the storage layer
 	DefaultEndpointReconcilerTTL = 15 * time.Second
 	// DefaultStorageEndpoint is the default storage endpoint for the lease controller
```

Comment:
This comment doesn't really make sense insofar as what it's really used for, and we should probably change the name of the constant too.

Reply:
Agreed.  What about EndpointPrefix?

Reply:
Actually, I think we can get rid of this const alltogether

Reply:
exactly